server/controllers/trustController.js

```diff
@@ -0,0 +1,43 @@
+const trustController = {};
+const pool = require('../database/database.js');
+const { check, validationResult } = require('express-validator');
+const config = require('../../config/config.js');
+const log = require("loglevel");
+log.setLevel("debug");
+const TrustModel = require("../models/TrustModel");
+const asyncHandler = require('express-async-handler');
+const expect = require("expect-runtime");
+
+const asyncUtil = fn =>
```

Comment:
This code was put here temporarily,  I extended the lib that we are using previously [express-async-handler](https://github.com/Abazhenov/express-async-handler#readme) , so we can easily handle error and response to the client the appropriate json data, like: 
{
  code: 400,
  message: 'invalid wallet name'
}